src/identifiers.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2021 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const ELEMENT_CLIENT_ID = "im.vector.web"; // root namespace for a lot of our settings
```

Comment:
Hmm, why not just be modern and put Element in the value?

Reply:
because we've historically expressed value in sticking to our Vector roots, even through all of Riot. I don't personally have an opinion.